client/lib/faux-plugins.js

```diff
@@ -232,13 +243,13 @@ module.exports = function(roomName) {
             time: msg.get('time'),
             messageId: msg.get('id'),
             youtubeId: match[1],
+            youtubeTime: parseYoutubeTime(match[3] || "0"),
```

Comment:
I think it would be cleaner do to something like `match[3] ? parseYouTubeTime(match[3]) : 0` here.